securedrop/wordlist

```diff
@@ -1012,7 +989,6 @@ chihuahua
 childcare
 childhood
 childish
-childless
```

Comment:
My reasoning on this one would a that a whistleblower could lose custody of a child as a result of their actions, and this word might be "triggering" in that regard.